fix_sonar.sh

```
@@ -0,0 +1,2 @@
+sed -i 's/Query q = entityManager.createNativeQuery(sql.toString()); \/\/ NOSONAR javasecurity:S3649/Query q = entityManager.createNativeQuery(sql.toString()); \/\/ NOSONAR javasecurity:S3649 -- Blocklist applied before concatenation/g' src/main/java/io/github/carlos_emr/carlos/commn/dao/EFormReportToolDaoImpl.java
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)